.gitlab-ci.yml

```diff
@@ -77,6 +77,7 @@ build_exatrkx:
     exclude:
       - build/**/*.o
       - build/bin/ActsIntegrationTest*
+    expire_in: 6 hours
```

Comment:
Could we keep this to maybe 1 day or at least 12 hours?
The GitLab jobs often take quite long to start, so I let them frequently run over night. But then the artifacts would be gone in the morning.